components/fieldlabel/index.css

```diff
@@ -116,54 +113,57 @@ governing permissions and limitations under the License.
   text-align: end;
 }
 
-/* Form: two column alignment via display: table */
+/********* Form *********/
 .spectrum-Form {
   --spectrum-tableform-item-block-spacing: var(--spectrum-spacing-300);
-  --spectrum-tableform-border-spacing: 0 var(--mod-tableform-item-block-spacing, var(--spectrum-tableform-item-block-spacing));
-  --spectrum-tableform-margin: calc(var(--mod-tableform-item-block-spacing, var(--spectrum-tableform-item-block-spacing)) * -1) 0;
 
-  display: table;
-  border-collapse: separate;
-  border-spacing: var(--mod-tableform-border-spacing, var(--spectrum-tableform-border-spacing));
-  margin: var(--mod-tableform-margin, var(--spectrum-tableform-margin));
+  margin: 0;
+  display: grid;
+  grid-template-columns: var(--mod-form-grid-template-columns, auto auto);
+  inline-size: var(--mod-form-inline-size, fit-content);
+  justify-content: start;
+
+  /* @deprecation --mod-tableform-item-block-spacing has been renamed to
+     --mod-form-item-block-spacing. The fallback will be removed in a future version. */
```

Comment:
A helpful comment. Is this how we usually let consumers know about deprecations?

Reply:
Yes, I'm following this suggestion as was mentioned in a Slack discussion. To add to the fallback stack, include a comment about deprecation, and a note in the commit message. Based on [an linked example from Dropzone](https://github.com/adobe/spectrum-css/blob/a0cf37bf5fa1093ab6fd6ae8244c9b93352907aa/components/dropzone/index.css#L117).